src/Tools/MicroBuild/run-pdbgit.ps1

```diff
@@ -40,6 +40,14 @@ try
     $count = 0
 
     foreach ($relativePath in $signToolData.sign | %{ $_.values }) {
+
+        # This is a DLL we sign but do not produce.  Hence it doesn't have a PDB
+        # to be source linked.
```

Comment:
I think it's wrong. We are not building it in the repo so we shouldn't be signing it. It's signed by the build queue of symreader-portable repo.